ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -195,7 +196,7 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
         );
         retryWatch.start(workflow.metadata.resourceVersion);
         return () => retryWatch.stop();
-    }, [workflow]);
+    }, [namespace, name]);
```

Comment:
this was a pretty nasty bug, every time we got a changed workflow, we re-established the watch, resulting in a hot loop